src/amo/pages/Addon/index.js

```diff
@@ -299,6 +299,34 @@ export class AddonBase extends React.Component {
     ) : null;
   }
 
+  renderDevCommentsCard = () => {
+    const { addon } = this.props;
+    const commentsProps = {};
+    let showComments = true;
+    let comments;
+
+    if (addon && addon.developer_comments) {
+      comments = addon.developer_comments;
+    }
+
+    if (!comments || comments === null) {
+      showComments = false;
+    }
+    commentsProps.dangerouslySetInnerHTML = sanitizeUserHTML(comments);
+
+    const showMoreCardName = 'AddonDevelopersComments';
+
+    return showComments ? (
+      <ShowMoreCard
+        className={showMoreCardName}
+        header={`Developer's comments`}
```

Comment:
We will need to translate this string so we should call `i18n.gettext()`:

```suggestion
        header={i18n.gettext(`Developer's comments`)}
```